spec/player_factory_spec.rb

```diff
@@ -61,6 +67,16 @@ def run_set_up
       expect(result[1].name).to eq("Hank")
     end
       
+    it "it calls validator.name_selection_valid? to check if the name is valid" do
```

Comment:
The test name here exposes too much of the method's implementation. One way to improve that is to remove the extra details.

For example: `it "validates names provided by user" do`.